docs/application/toc_all.md

```diff
@@ -90,6 +90,8 @@
 ##### [Switch](/application/dotnet/guides/nui/nui-components/Switch.md)
 ##### [Tab](/application/dotnet/guides/nui/nui-components/Tab.md)
 ##### [Toast](/application/dotnet/guides/nui/nui-components/Toast.md)
+##### [CheckBox](/application/dotnet/guides/nui/nui-components/CheckBox.md)
+##### [RadioButton](/application/dotnet/guides/nui/nui-components/RadioButton.md)
```

Comment:
@mijin85cho,

Would it be good to follow alphabetical order here as well? Is there a reason why it was not followed. Please let us know your views.

Reply:
Thank you for the clarification. I agree with it.

Reply:
fixed